docs/source/tutorials/token_classification/utils.py

```diff
@@ -0,0 +1,131 @@
+import string 
```

Comment:
These functions don't seem super specific to CoNLL right?  If not, then let's add them to: `cleanlab/token_classification/utils.py` instead of in the tutorials folder. Strictly speaking the tutorials folder ideally only contains tutorial notebooks and nothing else.  If you need some helper functions inside the tutorial notebook, they should ideally be defined within the tutorial notebook so it is more self-contained (eg. when viewed on docs.cleanlab.ai or when run in google colab).